content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
+
+A typical interview process at Incubyte is divided into 5 stages:
+
+1. Application
+2. Pre-required learning
+3. Call with the People Success Team
+4. Technical Discussion Round
+5. Software Craft Round
+
+## Pre-required Learning
+After you send in your application, and your resume has been screened by the People Success Team, you will receive an email that lists a few concepts and short videos. This step exists to align expectations through the interview process. This pre-requisite learning email will equip you with resources that set you up for success and.. we give you unlimited time to go through it!
+
+Here are some of the things we recommend: 
+- [Test-Driven Development](https://www.youtube.com/watch?v=qkblc5WRn-U)
+- [Software Craftsmanship](https://www.youtube.com/watch?v=c07uQGBZl0A)
+- [Clean Code](https://www.youtube.com/watch?v=zV079g7Irks&feature=emb_imp_woyt)
+
+This step also shines light on other skills needed to clear the interviews. For example, knowing common Git operations, hands-on knowledge of your IDE (with shortcuts), and an understanding of a unit testing framework to name a few!
+
+## Call with the People Success Team
+In this step, we aim to align on the role you are looking for and what we have to offer and see if there is synergy there. Being remote-first also means that it is important that we align not just on tech; but also on culture, attitude, and core values! Our People Success Team are here to listen to your goals and help answer any questions you may have about the Company. Ask away!
+
+## Technical Discussion with our existing team members
+A 30-minute discussion around your technical background and journey so far. You can expect the conversation to revolve around languages, frameworks, and databases you have worked with; your prior experiences, projects, etc. This is for us to get introduced to the depth and breadth of your technical prowess. Here it is important that you have a good Wi-Fi connection since we'd want to have your video on during the discussion.
+
+## Software Craft Round
+Congratulations! You have gotten to this round. Here is where all the fun happens. You will fire up your IDE with the language of your choice and a simple Hello World program pre-installed and get ready to get your hands dirty. We will solve a short Kata together as a team! Consider this round as a pairing exercise, in which an Incubyte developer will partner with you and act as a navigator. It is fine to look stuff up on the internet. We understand that having Maven or Gradle dependency coordinate on top of the code is difficult. (We google things while working on production projects ALL the time!)
```

Comment:
"Hello, world program pre-installed" does not sound right. 
Should it be "A project and IDE setup, able to execute a simple hello world program"